packages/code-studio/src/plugins/PluginUtils.jsx

```diff
@@ -7,17 +7,19 @@ const log = Log.module('PluginUtils');
 
 class PluginUtils {
   /**
-   * Load a component plugin either specified in the REACT_APP_COMPONENT_TABLE_PLUGINS environment variable, or from the server if it's not internal.
+   * Load a component plugin either specified in the VITE_INTERNAL_COMPONENT_PLUGINS environment variable, or from the server if it's not internal.
    * @param {string} pluginName Name of the table plugin to load
    * @returns {JSX.Element} A lazily loaded JSX.Element from the plugin
    */
   static loadComponentPlugin(pluginName) {
     if (
-      process.env.REACT_APP_INTERNAL_COMPONENT_PLUGINS.split(',').includes(
+      import.meta.env.VITE_INTERNAL_COMPONENT_PLUGINS.split(',').includes(
```

Comment:
Tested with internal plugins?

Reply:
Works. Although the way to attach plugins is not good in the Python API (you have to get the underlying j_table and attach attributes)